_commands/toys/cowsay.md

```diff
@@ -3,22 +3,62 @@
 
 Cowsay
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+'cowsay' is a program that shows a ASCII picture of a cow saying a message inside the terminal.
 
-<!-- minimal example -->
-~~~ bash
-*short* example of command usage and output
+To use 'cowsay', simply type:
+
+~~~bash
+$ cowsay "your message"
 ~~~
 
-<!--more-->
+This script uses ascii art to show a cow in the terminal saying anything you tell it to say.
 
-### Useful Options / Examples
+`cowsay`  is not built-in to your terminal and in order to use it, you have to install it to your machine. 
 
-#### Example command
+If you are using a linux machine, do the following steps:
+
+~~~bash
+$sudo apt-git install cowsay
+~~~
 
-##### Break it down
+### Useful Commands / Examples 
+- `-f cowfile` - specifies from which file to load ascii art (cowsay -f vader.cow - turns the cow into Cowth Vader) 
```

Comment:
nit: the dash after the initial option here can be removed since you're using the code formatting.

Also, the "cowsay -f vader.cow" later in the line can be included in the code formatting. `code -f vader.cow`

Reply:
I think I fixed up the stuff like you asked, but I removed the files and now the travis thing isn't working anymore. Any advice on that? Thanks